enclave_wrangler/objects_api.py

```diff
@@ -648,6 +658,16 @@ def fetch_cset_expression_item(object_id: int, retain_properties_nesting=False)
     return fetch_object_by_id('OmopConceptSetVersionItem', object_id, 'itemId', retain_properties_nesting)
 
 
+def update_objects_in_db(
```

Comment:
## Update function (enclave objects)
Converts them into table objects and then kicks off updates.